src/oid.rs

```diff
@@ -1,74 +1,68 @@
-/// pkcs-9-at-extensionRequest in RFC 2985
+/// pkcs-9-at-extensionRequest in <https://www.rfc-editor.org/rfc/rfc2985#appendix-A>
 pub const OID_PKCS_9_AT_EXTENSION_REQUEST: &[u64] = &[1, 2, 840, 113549, 1, 9, 14];
 
-/// id-at-countryName in RFC 5280
+/// id-at-countryName in <https://www.rfc-editor.org/rfc/rfc5280#appendix-A>
```

Comment:
How is this rendered? Having the full URL visible? Then maybe a link like ```[`RFC 5280`](https://www.rfc-editor.org/rfc/rfc5280#appendix-A)``` would be better. Note how the URLs were non-doc comments.

Reply:
Switched to titled links.